src/test/java/com/egov/business-services/tests/egfMasterChartOfAccountUpdate.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Chart of account Update
+Background:
+  * call read('../../business-services/tests/egfMasterChartOfAccountCreate.feature@ChartOfAccountCreate_01')
+  * def jsUtils = read('classpath:jsUtils.js')
+  * def commonConstant = read('../../common-services/constants/genericConstants.yaml')
+  * def egfMasterChartOfAccountConstant = read('../../business-services/constants/egfMasterChartOfAccount.yaml')
+  * def invalidTenantId = ranString(3)
+
+@ChartOfAccountUpdate_01  @chartOfAccountUpdate  @chartOfAccount  @positive  
+Scenario: Verify updating chart of accounts through API call
+# Update the glcode
+* def glcode = glcode
+* call read('../../business-services/preTests/egfMasterChartOfAccount.feature@updateAccountSuccessfully')
+* match chartOfAccountUpdateResponseBody.chartOfAccounts.length == '##[_ > 0]'
```

Comment:
Please mention what exactly you are trying to verify here. Put some comment on it.